geode-core/src/main/java/org/apache/geode/cache/client/internal/QueueManagerImpl.java

```diff
@@ -373,8 +373,8 @@ private void endpointCrashed(Endpoint endpoint) {
                   ? (deadConnection.getUpdater().isPrimary() ? "Primary" : "Redundant")
                   : "Queue",
                   endpoint});
-      scheduleRedundancySatisfierIfNeeded(0);
       deadConnection.internalDestroy();
+      scheduleRedundancySatisfierIfNeeded(0);
```

Comment:
I'm glad you did this. I wanted to reverse these two method calls for GEODE-10009.